doc_src/cmds/fish.rst

```diff
@@ -13,17 +13,17 @@ Synopsis
 Description
 -----------
 
-``fish`` is a command-line shell written mainly with interactive use in mind. This page briefly describes the options for invoking fish. The :ref:`full manual <intro>` is available in HTML by using the :ref:`help <cmd-help>` command from inside fish, and in the `fish-doc(1)` man page. The :ref:`tutorial <tutorial>` is available as HTML via ``help tutorial`` or in `fish-tutorial(1)`.
+fish is a command-line shell written mainly with interactive use in mind. This page briefly describes the options for invoking fish. The :ref:`full manual <intro>` is available in HTML by using the :ref:`help <cmd-help>` command from inside fish, and in the `fish-doc(1)` man page. The :ref:`tutorial <tutorial>` is available as HTML via ``help tutorial`` or in `fish-tutorial(1)`.
 
 The following options are available:
 
 - ``-c`` or ``--command=COMMANDS`` evaluate the specified commands instead of reading from the commandline
 
 - ``-C`` or ``--init-command=COMMANDS`` evaluate the specified commands after reading the configuration, before running the command specified by ``-c`` or reading interactive input
 
-- ``-d`` or ``--debug=CATEGORY_GLOB`` enables debug output and specifies a glob for matching debug categories (like ``fish -d``). Defaults to empty.
+- ``-d`` or ``--debug=DEBUG_CATEGORIES`` enable debug output and optionally specify a pattern for matching debug categories. See :ref:`Debugging <debugging-fish>` below for details.
```

Comment:
Maybe it would be more natural if we accepted multiple `-d` arguments, each with just one glob. Making that change is probably unnecessary, though.

Reply:
As you may have noticed, the `--debug` argument _is_ optional. If you only give `--debug` you get the default categories. You can also pass the legacy arguments (number 0-5, e.g. `-d2`), although that seems to be pointless since most debug statements go through FLOG now.

Reply:
Fixed the description. As for accepting multiple `-d` arguments, I'd rather not change the contract at this point. And having examples helps.